client/my-sites/github-deployments/deployments/styles.scss

```diff
@@ -209,3 +212,24 @@
 		padding: 16px;
 	}
 }
+
+.github-deployments {
+	.header-cake {
+		gap: 32px;
+		font-family: $brand-text;
+	}
+	.header-cake__title {
+		flex: none;
+		color: var(--studio-gray-80);
+		font-size: $font-body;
+		padding: 0;
+	}
+	.header-cake__back {
+		color: var(--studio-gray-60);
+		font-size: $font-body-small;
+		svg {
```

Comment:
Is it possible to make the icon smaller and closer to the "Back" text?

| Design | Current |
| - | - |
| ![image](https://github.com/Automattic/wp-calypso/assets/13596067/ebf50ff3-108a-4390-a1ee-f41e5d57b8c2) | ![image](https://github.com/Automattic/wp-calypso/assets/13596067/ec7531a7-5488-4ca2-b47b-c5b9a0dc6dcf) |

Reply:
The simplest way I found here is to set the width in css with `!important`.